pandas/_config/localization.py

```diff
@@ -100,10 +100,13 @@ def _valid_locales(locales, normalize):
 def _default_locale_getter():
     try:
         raw_locales = subprocess.check_output(["locale -a"], shell=True)
-    except subprocess.CalledProcessError as e:
-        raise type(e)(
```

Comment:
apparently in the status quo this ends up raising TypeError because we are calling the CalledProcessError constructor incorrectly

Reply:
Do we even need this at all then? Seems like it wasn't doing what was expected before so maybe can remove altogether?

Reply:
I was wondering this myself.  Will change.

Reply:
changed per request (and green)